catalyst/utils/metrics/mrr.py

```diff
@@ -0,0 +1,40 @@
+"""
+MRR metric.
+"""
+
+import torch
+
+
+def mrr(outputs: torch.Tensor, targets: torch.Tensor):
+
+    """
+    Calculate the MRR score given model ouptputs and targets
+    Args:
+        outputs [batch_size, slate_length] (torch.Tensor): model outputs, logits
+        targets [batch_szie, slate_length] (torch.Tensor): ground truth, labels
+    Returns:
+        mrr (float): the mrr score
+    """
+    outputs = outputs.clone()
+    targets = targets.clone()
```

Comment:
why do you need `clone`?

Reply:
Tried to follow the  'shared clones', which is a common pattern in torch. But it seems it not so necessary here.